packages/paste-core/components/form/__tests__/select/option.test.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {render} from 'react-dom';
+import {render as testRender, cleanup} from '@testing-library/react';
+import {axe} from 'jest-axe';
+import {Option} from '../../src';
+import {createAttributeMap} from '../../test-utils';
+
+interface MockOptionProps extends React.OptionHTMLAttributes<HTMLOptionElement> {
+  suffix?: string;
+  value?: string;
+}
+
+const MockOption: React.FC<MockOptionProps> = ({suffix = 'test', ...props}): React.ReactElement => {
+  return (
+    <Option data-testid={`option-${suffix}`} value={'option-1'} {...props}>
+      Option 1
+    </Option>
+  );
+};
+
+describe('Form | Option', () => {
+  afterEach(cleanup);
```

Comment:
You can drop the `afterEach(cleanup)` on the tests. Jest handles that: https://testing-library.com/docs/react-testing-library/api#cleanup